dmwmclient/datasvc.py

```diff
@@ -494,3 +494,172 @@ async def missingfiles(self, human_readable=None, **params):
+                                "Size": _block["bytes"],
+                                "Time_create": _block["time_create"],
+                                "Number_of_files": _block["files"],
+                                "Time_update": _block["time_update"],
+                                "Destination": _destination["name"],
+                                "custodial": _latency["is_custodial"],
+                                "last_suspend": _latency["last_suspend"],
+                                "last_replica": _latency["last_replica"],
+                                "time_subscription": _latency["time_subscription"],
+                                "block_closed": _latency["block_close"],
+                                "latency": _latency["latency"],
+                            }
+                        )
+            df = pandas.json_normalize(out)
+            return format_dates(
+                df,
+                [
+                    "Time_update",
+                    "last_suspend",
+                    "last_replica",
+                    "time_subscription",
+                    "block_closed",
+                    "Time_create",
+                ],
+            )
+        elif human_readable is True:
+            for _block in resjson["phedex"]["block"]:
+                for _destination in _block["destination"]:
+                    for _latency in _destination["latency"]:
+                        out.append(
+                            {
+                                "Block": _block["name"],
+                                "Block ID": _block["id"],
+                                "Dataset": _block["dataset"],
+                                "Size": _block["bytes"],
+                                "Time Create": _block["time_create"],
+                                "Number of files": _block["files"],
+                                "Time Update": _block["time_update"],
+                                "Destination": _destination["name"],
+                                "custodial": _latency["is_custodial"],
+                                "Last Suspend": _latency["last_suspend"],
+                                "Last Replica": _latency["last_replica"],
+                                "Time Subscription": _latency["time_subscription"],
+                                "Block Closed": _latency["block_close"],
+                                "Latency": _latency["latency"],
+                            }
+                        )
+            df = pandas.json_normalize(out)
+            return format_dates(
+                df,
+                [
+                    "Time Update",
+                    "Last Suspend",
+                    "Last Replica",
+                    "Time Subscription",
+                    "Block Closed",
+                    "Time Create",
+                ],
+            )
+
+    async def BlockReplicaSummary(self, human_readable=None, **params):
```

Comment:
Here again the capitalization differs from the API. Although I agree that at least some capitalization might be nice, maybe consider using camelCase?

Reply:
I'll let it be as in the api if that's ok for you. Not a fan of camelCase, tbh.